### PR TITLE
Small updates on Cleanup wildcard in module part (Overall re-checking at 81X)

### PR DESCRIPTION
**Pull request by jeongeun**
Overall Re-Check at 81X :  these 8 workflows're considered to cleanup the wildcards in module parts for EventContent.
1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT,
1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4,
1330.0_ZMM_13+ZMM_13INPUT+DIGIUP15+RECOUP15+HARVESTUP15,
134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_2016+RECODR2_2016reHLT+HARVESTDR2,
135.4_ZEE_13+ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS,
140.53_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI,
4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC,
8.0_BeamHalo+BeamHaloINPUT+DIGICOS+RECOCOS+ALCABH+HARVESTCOS,

### CODE REVIEW

**Comment by mmusich** [comment]
please test

File: Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalGammaJet_Output_cff.py [hunk 1]
```diff
@@ -8,12 +8,37 @@
...
+                 'keep recoSuperClusters_multi5x5SuperClustersWithPreshower_*_*',
+                 'keep recoSuperClusters_particleFlowEGamma_*_*',
+                 'keep recoSuperClusters_uncleanedHybridSuperClusters_*_*',
+                 'keep recoSuperClusters_uncleanedOnlyCorrectedHybridSuperClusters_*_*',
+                 'keep recoSuperClusters_uncleanedOnlyCorrectedMulti5x5SuperClustersWithPreshower_*_*',
+                 'keep recoSuperClusters_uncleanedOnlyMulti5x5SuperClustersWithPreshower_*_*',
+                 'keep recoSuperClusters_multi5x5SuperClustersCleaned_*_*',
+                 'keep recoSuperClusters_multi5x5SuperClustersUncleaned_*_*',
+                 'keep recoSuperClusters_multi5x5SuperClusters_*_*',
+                 'keep recoSuperClusters_particleFlowSuperClusterECAL_*_*',
```
Comment (slava77) [inline Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalGammaJet_Output_cff.py]: This removed the following branches:
```
recoSuperClusters_reducedEgamma_reducedSuperClusters_*.
recoPhotonCores_reducedEgamma_reducedGedPhotonCores_*.
recoSuperClusters_mergedSuperClusters__*.
```
based on modified test derived from 134.911, adding HcalCalDijets+HcalCalGammaJet to step3.

@jeongeun 
Please make sure that your test config is running relevant ALCAs, if they are modified.

**Comment by slava77** [comment]
@jeongeun 
other than the issue mentioned in https://github.com/cms-sw/cmssw/pull/14406#discussion-diff-64278071
I think the rest of the PR is OK

**Comment by gpetruc** [comment]
Checking, as it seems to me this will drop the `slimmedJetsPuppi`.

May I ask why do we need this in the first place?

**Comment by gpetruc** [comment]
Sorry, you can ignore my last message, the change was only for the GenJets for which there's no Puppi. So, the change is fine for me.

**Comment by slava77** [comment]
the goal of changes in this PR is to not use wildcards in module names.
This is safer than making sure by other means that unneeded modules are not loaded in the configuration and end up running.
We know that there are about 500 modules (about 30% of the total; a large fraction is PAT-related) that are loaded in standard configs and may end up executed in unscheduled mode if there is a too inclusive keep statement.

**Comment by slava77** [comment]
@jeongeun 
ping
this is to remind of the issue found in one of the changed fiiles:
https://github.com/cms-sw/cmssw/pull/14406#discussion_r64278071

**Comment by slava77** [comment]
@jeongeun 
the command I used for step3 in 134.911 was:
```
cmsDriver.py step3  --conditions auto:run2_data_relval -s RAW2DIGI,L1Reco,RECO,EI,PAT,\
ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBias+EcalESAlign+EcalCalZElectron\
+EcalUncalZElectron+EcalCalWElectron+EcalUncalWElectron+HcalCalDijets+HcalCalGammaJet,\
DQM:@standardDQM+@miniAODDQM --runUnscheduled  --process reRECO --data  --era Run2_2016\
 --eventcontent RECO,MINIAOD,DQM --hltProcess reHLT --scenario pp\
 --datatier RECO,MINIAOD,DQMIO\
 --customise Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2016\
 -n 100  --filein file:step2.root  --fileout file:step3.root  >& step3.log &
```
(this was all a one-line, reformatted here for visibility)
The modifications from the original 134.911 command was the additions of affected ALCA sequences after ALCA:

**Pull request closed without merging**